backend/app/api/admin_routes/document.py

```diff
@@ -1,19 +1,62 @@
-from fastapi import APIRouter, Depends
+from fastapi import APIRouter, Depends, Query
 from fastapi_pagination import Params, Page
 
 from app.api.deps import SessionDep, CurrentSuperuserDep
 from app.repositories import document_repo
-from app.models import Document
+from app.models import Document, DocIndexTaskStatus
+
+from datetime import datetime
+
+from app.types import MimeTypes
 
 router = APIRouter()
 
 
 @router.get("/admin/documents")
 def list_documents(
     session: SessionDep,
-    user: CurrentSuperuserDep,
+    # user: CurrentSuperuserDep,
     params: Params = Depends(),
-    query: str | None = None,
+    source_uri: str | None = Query(
+        None,
+        description="[Fuzzy Match] source URI field, will search for the source URI that contains the given string."
+    ),
     data_source_id: int | None = None,
+    created_at_start: datetime | None = None,
+    created_at_end: datetime | None = None,
+    updated_at_start: datetime | None = None,
+    updated_at_end: datetime | None = None,
+    last_modified_at_start: datetime | None = None,
+    last_modified_at_end: datetime | None = None,
+    name: str | None = Query(
+        None,
+        description="[Fuzzy Match] name field, will search for the name that contains the given string."
+    ),
+    mime_type: MimeTypes | None = None,
+    index_status: DocIndexTaskStatus | None = None,
+    language: str | None = Query(
+        None,
+        description="[Exact Match] meta.language field, for example: 'en', 'zh', etc."
+    ),
+    product: str | None = Query(
```

Comment:
Why there is a "product" field here, and together with "tidb", it is too specific.

Reply:
I used meta fields here. We shouldn't use meta data here right?